metricbeat/docs/modules/system.asciidoc

```diff
@@ -94,39 +94,39 @@ metricbeat.modules:
 
 The following metricsets are available:
 
-* <<metricbeat-metricset-system-core,core>>
+* <<metricbeat-metricset-system-diskio,diskio>>
```

Comment:
Not sure why these changes show up here as this should not have changed. I assume it is based on an old version of master.